### ISSUE TITLE
Fail on ubuntu 16.04 64bit

### ISSUE THREAD

**Issue by ookangzheng** [fardog/secureoperator#10]
my Go version is 1.8 version
<img width="1448" alt="screen shot 2017-05-28 at 17 51 37" src="https://cloud.githubusercontent.com/assets/6301735/26527802/6910e244-43ce-11e7-9ab1-a6f11cc0ba8f.png">

**Comment by fardog** [comment]
I'm unsure what would cause this issue; have you had problems with any other go software installing? i've updated the CI tests to use 1.8 (see #12) which is working fine, and I build locally with 1.8 (on Arch Linux) and have had no issues. If you look at the [go docs for `http.Transport`](https://golang.org/pkg/net/http/#Transport) you can see that those fields are present, so I'm unsure why that error is occurring for you. I don't currently have Ubuntu installed to test with.

you may have better luck downloading a [release](https://github.com/fardog/secureoperator/releases/) directly. i'm going to leave this issue open in case any other ubuntu users experience issues, they can chime in.

**Comment by ookangzheng** [comment]
if I download the release files, how can I run it? 
just like ./secureoperator ?

**Comment by fardog** [comment]
yes; once you download the release, you can just run it directly. on most unix/linux systems, you will first need to make it executable with a `chmod +x secureoperator` then run it with `./secureoperator` (or whatever the filename you downloaded is)
